### PR TITLE
add badges

### PR DESCRIPTION
**Pull request by anowakiteo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,12 +11,18 @@ and the Flutter guide for
 [developing packages and plugins](https://flutter.dev/developing-packages).
 -->
 
+<div align="center">
```
Comment (tomkad99) [inline README.md:14]: Many other packages have badges just under the logo. I think it looks like aesthetically that logo is first and it naturally points user to the elements below icon

Comment (StolarczykJakub) [reply README.md:14]: I agree. Lets move it below logo.

**Review by StolarczykJakub — CHANGES_REQUESTED** [review]
Move badges below logo.

File: README.md (hunk 1 above)
Comment (anowakiteo) [reply README.md:1]: right, done

**Review by tomkad99 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,102 +1,107 @@
...
 [![codecov](https://codecov.io/gh/Iteo/hooked_bloc/branch/main/graph/badge.svg)](https://codecov.io/gh/Iteo/hooked_bloc)
+&nbsp;
+[![pub package](https://img.shields.io/pub/v/hooked_bloc.svg)](https://pub.dartlang.org/packages/hooked_bloc) &nbsp;
+[![stars](https://img.shields.io/github/stars/Iteo/hooked_bloc.svg?style=flat&logo=github&colorB=deeppink&label=stars)](https://github.com/Iteo/hooked_bloc)
+&nbsp;
+[![GitHub license](https://img.shields.io/github/license/Iteo/hooked_bloc)](https://github.com/Iteo/hooked_bloc/blob/main/LICENSE) &nbsp;
+
+</div>
+
+---
```
Comment (tomkad99) [inline README.md:17]: I think this duplicates divider with `#Hooked bloc` title

**Review by tomkad99 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (anowakiteo) [reply README.md:17]: done

**Review by StolarczykJakub — APPROVED** [review]

**Pull request merged**